cfe_internal/CFE_cfengine.cf

```diff
@@ -31,6 +31,14 @@ bundle agent cfe_internal_management
 
     am_policy_hub.enterprise::
 
+      "hub" -> { "Mission Portal", "CFEngine Enterprise" }
+        usebundle => cfe_internal_purge_scheduled_reports_older_than_days( $(def.purge_scheduled_reports_older_than_days) ),
+        ifvarclass => "enable_cfe_internal_purge_scheduled_reports_older_than_days",
```

Comment:
Don't think ifvarclass is necessary, assume almost everyone would want this (and they can adjust the threshold in days).